components/expenses/ExpenseInviteNotificationBanner.js

```diff
@@ -0,0 +1,153 @@
+    if (userExists) {
+      const redirect = window.location.pathname + window.location.search;
+      return await signin({
+        user: { email: createdUser.email },
+        redirect,
+        websiteUrl: getWebsiteUrl(),
+      });
+    } else {
+      throw Error('User does not exist.');
+    }
+  });
+  const success = !error && data?.success;
+  return (
+    <StyledButton
+      buttonStyle={success ? 'successSecondary' : 'primary'}
+      buttonSize="tiny"
+      mr={1}
+      loading={loading}
+      disabled={success}
+      onClick={call}
+    >
+      {success ? (
+        <FormattedMessage id="ResendEmailSuccessful" defaultMessage="Check your inbox!" />
+      ) : (
+        <FormattedMessage id="ResendEmail" defaultMessage="Resend email" />
+      )}
+    </StyledButton>
+  );
+};
+
+ResendSignInEmailButton.propTypes = {
+  createdUser: PropTypes.object.isRequired,
+};
+
+const ExpenseInviteNotificationBanner = props => {
+  return (
+    <StyledCard py={3} px="26px" mb={4} borderStyle={'solid'}>
+      <Flex>
+        <PidgeonIllustration />
+        <Flex ml={[0, 2]} maxWidth="448px" flexDirection="column">
+          <H4 mb="10px" fontWeight="500">
+            {props.createdUser ? (
+              <FormattedMessage id="VerifyEmailAddress" defaultMessage="Verify your email address" />
+            ) : (
+              <FormattedMessage id="InviteOnItsWay" defaultMessage="Your invite is on its way" />
+            )}
+          </H4>
+          <P lineHeight="20px">
+            {props.createdUser ? (
+              <FormattedMessage
+                id="VerifyEmailInstructions"
+                defaultMessage="An email has been sent to {email} with a link to verify your account. if you have not
+                      received the email after a few minutes, please check your spam folder."
+                values={{
+                  email: props.createdUser?.email || props.expense.draft?.payee?.name,
+                }}
+              />
+            ) : (
+              <FormattedMessage
+                id="Expense.InviteIsOnItsWay.Description"
+                defaultMessage="An email has been sent to {email} with the invitation to submit this expense. Only after this person validates his/her email and submit it, the expense would appear in the list."
+                values={{
+                  email: props.expense.draft?.payee?.email || props.expense.draft?.payee?.name,
+                }}
+              />
+            )}
+          </P>
+          <Box mt="10px">
+            {props.createdUser ? (
+              <ResendSignInEmailButton createdUser={props.createdUser} />
+            ) : (
+              <ResendDraftInviteButton expense={props.expense} />
+            )}
+            <StyledLink href="mailto:support@opencollective.com" buttonStyle="standard" buttonSize="tiny">
+              <FormattedMessage id="ContactSupport" defaultMessage="Contact Support" />
```

Comment:
We already have a string for this one

```suggestion
              <FormattedMessage id="error.contactSupport" defaultMessage="Contact Support" />
```